iree/hal/cuda/cuda_headers.h

```diff
@@ -8,5 +8,6 @@
 #define IREE_HAL_CUDA_CUDA_HEADERS_H_
 
 #include "cuda.h"  // IWYU pragma: export
+#include "driver_types.h"
```

Comment:
you'll need to remove this (only cuda.h exists)

Reply:
but the `cudaMemcpyKind` I used for `cuMemcpyAsync` will be missing if I don't include it, any suggestions?